spec/views/events/index.html.erb_spec.rb

```diff
@@ -6,7 +6,7 @@
   it 'renders a list of events' do
     render
 
-    assert_select 'tr>td', text: 'Hashtag'.to_s, count: 2
+    assert_select 'tr>td', text: 'Hash_tag'.to_s, count: 2
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.